src/plugins/enzymes/src/FindEnzymesDialog.h

```diff
@@ -134,6 +134,14 @@ class EnzymeTreeItem : public QTreeWidgetItem {
     EnzymeTreeItem(const SEnzymeData& ed);
     SEnzymeData enzyme;
     bool operator<(const QTreeWidgetItem& other) const;
+
+private:
+    enum class Ns {
```

Comment:
nit: move private stuff into C++ if possible: this way you won't slow down a recompilation in case it is changed.

Reply:
OK